src/scripts/components/Button.jsx

```diff
@@ -61,20 +76,23 @@ class Button extends React.Component {
       'button--block': block,
       'button--icon': iconOnly,
       [UtilitySystem.config.classes.active]: active,
+      [UtilitySystem.config.classes.loading]: loading,
     });
 
     let markup = '';
 
     if (route) {
       markup = (
-        <Link to={route} className={classes} onClick={this.handleClick} {...opts} title={this.props.title}>
+        <Link to={route} className={classes} onClick={this.handleClick} disabled={disabled || loading} {...opts} title={this.props.title}>
           <span className="button__text-wrapper">{this.props.children}</span>
+          {this.loadingRender()}
```

Comment:
So instead of putting a ternary in the loadingRender function itself, you also have the option of writing it like this:

```javascript
        <Link to={route} className={classes} onClick={this.handleClick} disabled={disabled || loading} {...opts} title={this.props.title}>
          <span className="button__text-wrapper">{this.props.children}</span>
          {loading && this.loadingRender()}
        </Link>
```

and removing the `this.props.loading ? <content /> : null` in your function. I don't care either way, but thought I would point it out :)